app/views/govuk_publishing_components/components/docs/contextual_sidebar.yml

```
@@ -112,3 +112,20 @@ examples:
             - content_id: "2f8b848d-23c8-4f42-a41a-df1f81c64d0f"
               title: "Exporting"
               phase: "live"
+
+  with_ukraine_cta:
```

Do we need this sort of thing here?
```suggestion
  with_ukraine_cta:
    description: For documents tagged to the Ukraine topical event we show a custom Ukraine Invasion call to action element.
```